### PR TITLE
[media] Media module test plan does not test widgets

### PR DESCRIPTION
**Pull request by maltheism**
## Brief summary of changes

- [x] Have you updated related documentation?

#### Testing instructions (if applicable)

1. Follow the TestPlan.md instructions for the media module.

#### Link(s) to related issue(s)

* Resolves #  (Reference the issue this fixes, if any.)
#6439

### CODE REVIEW

File: modules/media/test/TestPlan.md [hunk 1]
```diff
@@ -90,3 +90,9 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
   4. Type text in the Uploaded by input and verify that the table gets filtered as you type.
   5. Select Language, Project, File Visibility, Instrument, Visit Label, Site and File type (independently and combined) to filter table further
     - The table should update and display filtered records accordingly
+
+**Test the Candidate Dashboard widget for Candidate Media**
+   1. Visit the Candidate Dashboard for a candidate (found inside the media module) and notice the widget titled Candidate Media.
+     - Check if the correct number of files are visible corresponding withe the column **File Name** for the candidate.
```
Comment (driusan) [inline modules/media/test/TestPlan.md:96]: Missing a check for ensuring the widget doesn't show up if the candidate has no media.

File: modules/media/test/TestPlan.md [hunk 2]
```diff
@@ -90,3 +90,10 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
   4. Type text in the Uploaded by input and verify that the table gets filtered as you type.
   5. Select Language, Project, File Visibility, Instrument, Visit Label, Site and File type (independently and combined) to filter table further
     - The table should update and display filtered records accordingly
+
+**Test the Candidate Dashboard widget for Candidate Media**
+   1. Visit the Candidate Dashboard for a candidate (found inside the media module) and notice the widget titled Candidate Media.
```
Comment (driusan) [inline modules/media/test/TestPlan.md:95]: I don't understand the "(found inside the media module)" parenthesis here..

Comment (maltheism) [reply modules/media/test/TestPlan.md:95]: @driusan sorry I'll remove the parenthesis.

File: modules/media/test/TestPlan.md [hunk 3]
```diff
@@ -90,3 +90,10 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
   4. Type text in the Uploaded by input and verify that the table gets filtered as you type.
   5. Select Language, Project, File Visibility, Instrument, Visit Label, Site and File type (independently and combined) to filter table further
     - The table should update and display filtered records accordingly
+
+**Test the Candidate Dashboard widget for Candidate Media**
+   1. Visit the Candidate Dashboard for a candidate found inside the media module and notice the widget titled Candidate Media.
+     - Check if the correct number of files are visible corresponding withe the column **File Name** for the candidate.
```
Comment (PapillonMcGill) [inline modules/media/test/TestPlan.md:96]:
```suggestion
     - Check if the correct number of files are visible corresponding with the column **File Name** for the candidate.
```

File: modules/media/test/TestPlan.md [hunk 4]
```diff
@@ -90,3 +90,10 @@ is selected, the file name should should start with [PSCID]\_[Visit Label]\_[Ins
...
+     - Check if the Last modified date is correct for the files of the candidate.
```
Comment (PapillonMcGill) [inline modules/media/test/TestPlan.md:97]:
```suggestion
     - Check if the **Last modified** date is correct for the files of the candidate.
```

**Pull request merged**